src/docker/DockerCompose.js

```diff
@@ -187,6 +187,9 @@ class DockerCompose {
       composeOptions: [
         '--env-file', `.env.${preset}`,
       ],
+      commandOptions: [
```

Comment:
I wouldn't pass `build` to all docker-compose commands. Let's pass this optionally to the `up` method?